magefile.go

```diff
@@ -45,7 +45,7 @@ import (
 
 func init() {
 	mage.SetBuildVariableSources(&mage.BuildVariableSources{
```

Comment:
Seems we need to use absolute paths, because the cross-build happens from inside the x-pack directory. (Or alternatively maybe we should change something in dev-tools to stop cd'ing into x-pack? Not sure why that's being done.)

```go
repoInfo, err := mage.GetProjectRepoInfo()
if err != nil {
        panic(err)
}
mage.SetBuildVariableSources(&mage.BuildVariableSources{
        BeatVersion: filepath.Join(repoInfo.RootDir, "version", "version.go"),
        GoVersion:   filepath.Join(repoInfo.RootDir, ".go-version"),
        DocBranch:   filepath.Join(repoInfo.RootDir, "docs/version.asciidoc"),
})
```